Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -0,0 +1,612 @@
+            "batch_id": batch_id,
+            "status": status
+        }
+
+        return self._http_request(
+            method='GET',
+            url_suffix=f"/orgs/{org_id}/tasks",
+            params=params,
+        )
+
+    def run_command(self, org_id, device_id, payload) -> List[Dict]:
+        params = {"o": org_id}
+
+        return self._http_request(
+            method='POST',
+            url_suffix=f"/servers/{device_id}/queues",
+            params=params,
+            data=payload,
+        )
+
+    def update_device(self, org_id, device_id, payload) -> List[Dict]:
+        params = {
+            "o": org_id
+        }
+
+        return self._http_request(
+            method='PUT',
+            url_suffix=f"/servers/{device_id}",
+            params=params,
+            data=payload,
+        )
+
+    def update_group(self, org_id, group_id, payload) -> List[Dict]:
+        params = {"o": org_id}
+
+        return self._http_request(
+            method='PUT',
+            url_suffix=f"/servergroups/{group_id}",
+            params=params,
+            data=payload,
+        )
+
+    def upload_vulnerability_sync_file(self, org_id, type, payload, files) -> List[Dict]:
+        return self._http_request(
+            method='POST',
+            url_suffix=f"/orgs/{org_id}/tasks/{type}/batches/upload",
+            data=payload,
+            files=files,
+        )
+
+''' HELPER FUNCTIONS '''
+
+def get_default_server_group_id(client: Client):
+    default_server_group_id = 0
+    groups = client.list_groups()
+
+    for group in groups:
+        if not group.get("name"):
+            default_server_group_id = group.get("id")
+            break
+
+    return default_server_group_id
+
+''' COMMAND FUNCTIONS '''
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        # TODO: ADD HERE some code to test connectivity and authentication to your service.
+        # This  should validate all the inputs given in the integration configuration panel,
+        # either manually or by using an API that uses them.
+        client.list_organizations()
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):  # TODO: make sure you capture authentication errors
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+def action_on_vulnerability_sync_batch(client: Client, args: Dict[str, Any]) -> CommandResults:
+    org_id = args.get('org_id', None) or DEFAULT_ORG_ID
```

Comment:
Nice. I was thinking that it would be nice to have these common strings as constants `'org_id'`, etc...good call on defaulting the getter though.

Reply:
Right, I see what you mean now with the `limit` comment.